posts/2024-12-16-jblog-two-year-celebration.adoc

```diff
@@ -0,0 +1,240 @@
+---
+layout: post
+title: The OpenLiberty Japanese blog has celebrated its second anniversary
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/una-tapa
+author_github: https://github.com/una-tapa
+seo-title: The OpenLiberty Japanese blog has celebrated its second anniversary
+seo-description: The OpenLiberty Japanese blog has reached its second anniversary since its launch in 2022. We will introduce the most popular articles to date and highlight the contributors who have supported us.
```

Comment:
```suggestion
seo-description: The Open Liberty Japanese blog has reached its second anniversary since its launch in 2022. We will introduce the most popular articles to date and highlight the contributors who have supported us.
```